`npm install freecodecamp-os` returns "Not found"

**Describe the bug**
[freecodecamp-os](https://registry.npmjs.org/freecodecamp-os) is not defined. 

**To Reproduce**
Steps to reproduce the behavior:
1. run `npm install freecodecamp-os`
2. You'll see this error:

```
npm ERR! code E404
npm ERR! 404 Not Found - GET https://registry.npmjs.org/freecodecamp-os - Not found
npm ERR! 404 
npm ERR! 404  'freecodecamp-os@*' is not in this registry.
npm ERR! 404 
npm ERR! 404 Note that you can also install from a
npm ERR! 404 tarball, folder, http url, or git url.
```

**Expected behavior**
The `README.md` says this should work... so I'd expect it to. 

**Desktop (please complete the following information):**
 - OS: [e.g. iOS]
 - Browser: [e.g. chrome, safari]
 - Version: [e.g. 22]

**Smartphone (please complete the following information):**
 - Device: iMac
 - OS: MacOS 13.0 (22A380)

--- a/README.md
+++ b/README.md
@@ -5,7 +5,7 @@
 1. Install `freecodecamp-os` in your project root:
 
 ```bash
-npm install freecodecamp-os
+npm install @freecodecamp/freecodecamp-os
 ```
 
 2. Add a `freecodecamp.conf.json` file to your project root:
@@ -68,3 +68,22 @@ npm install freecodecamp-os
 ```
 
 3. Add all the necessary files to your project (_use this repo as a reference_)
+
+## How this Works
+
+This repo consists of:
+
+- `.freeCodeCamp/` - the directory that is published to NPM
+  - `client/` - SPA showcasing the curricula content
+  - `tooling/` - directory containing Nodejs scripts to run a server serving the client
+- `.logs/` - directory containing files with terminal log data
+- `bash/` - directory for bash configuration
+- `client/` - directory for assets used in `.freeCodeCamp/client/`
+- `config/` - directory for project configuration and state
+- `curriculum/` - directory for curriculum content
+  - `locales/` - directory for different language versions of the curriculum
+- `tooling/` - directory for Nodejs scripts integrated with `.freeCodeCamp/tooling/` but specific to the curriculum
+
+The course content served on the client is written in Markdown files. The lesson/project tests run on the Nodejs server are written in the same markdown files.
+
+The `freecodecamp.conf.json` file is used to configure the course, and define the actions taken by the [freeCodeCamp - Courses](https://marketplace.visualstudio.com/items?itemName=freeCodeCamp.freecodecamp-courses) extension. The `freecodecamp-os` package version used should match the minor version used for the extension. This is configured in the `.devcontainer/devcontainer.json` file.

